tests/common/http_mocked_fixtures.py

```diff
@@ -197,6 +197,39 @@ def http_mocked_lpar(request, http_mocked_cpc_classic):  # noqa: F811
     return lpar
 
 
+@pytest.fixture(
+    scope='module'
+)
+def http_mocked_partition(request, http_mocked_cpc_dpm):  # noqa: F811
+    # pylint: disable=unused-argument,redefined-outer-name
+    """
+    Pytest fixture representing a HTTP-mocked Partition on a CPC in DPM mode.
+
+    Its CPC object can be accessed as the parent object.
+
+    A test function parameter using this fixture resolves to a
+    :class:`~zhmcclient.Partition` object that has only a
+    minimal set of properties in the object (those that are returned with the
+    'List Partitions of CPC' operation).
+    """
+
+    with requests_mock.mock() as m:
+        uri = http_mocked_cpc_dpm.uri + '/partitions'
+        m.get(uri, status_code=200, json={
+            'partitions': [
+                {
+                    'object-uri': '/api/partitions/part-id-1',
```

Comment:
minor: we often resort to fixed uuids for such uris, such as 00000000-1111-2222-3333-444444444444

Reply:
Sure, but I want to indicate in the string that this is a faked URI. We do that in many tests and in the mock support.